ansible_ai_connect/ai/api/telemetry/schema2.py

```diff
@@ -82,3 +83,21 @@ class AnalyticsProductFeedback:
     timestamp: str = field(
         default=Factory(lambda self: timezone.now().isoformat(), takes_self=True)
     )
+
+
+@frozen
+class OneClickTrialPlan:
+    created_at: str = field(validator=validators.instance_of(str), converter=str, default="")
+    expired_at: str = field(validator=validators.instance_of(str), converter=str, default="")
+    is_active: bool = False
+    name: str = field(validator=validators.instance_of(str), converter=str, default="")
+    id: int = 0
+
+
+@frozen
+class OneClickTrialStarted:
+    plans: list[OneClickTrialPlan] = field(factory=list, validator=validators.min_len(1))
+    rh_user_org_id: int = field(validator=validators.instance_of(int), converter=int, default=0)
+    timestamp: str = field(
+        default=Factory(lambda self: timezone.now().isoformat(), takes_self=True)
```

Comment:
For the record, you have an UTC datetime that you convert to a naive date string. This may be problematical if it's not properly handled as such later.

You can add a trailing `Z` to the string to clarify that this is indeed an UTC time, e.g:

```
>>> datetime.now(timezone.utc).strftime('%Y-%m-%dT%H:%M:%SZ')
'2024-09-30T15:10:06Z'
```

Reply:
Hey @goneri , as far as I understand calling the formatter, when no parameteres, already does that, as I see this results on Segment:
`
analytics.track('4e5221f7-beb2-41d3-a731-f2a89a188e39', 'OneClickTrial Started', {
    'plans': [
        {
            'created_at': '2024-09-27T21:38:47.165Z',
            'expired_at': '2024-12-26T21:38:47.165Z',
            'is_active': true,
            'name': 'trial of 90 days',
            'id': 1
        }
    ],
    'rh_user_org_id': 17329057,
    'timestamp': '2024-09-27T23:38:47.172Z'
})

So they already contain the trailing 'Z'. 

Please let me know if I am missing something else, or that's already looking fine.

Thx!
`

Reply:
How awesome. Thanks!